olad/Makefile.mk

```diff
@@ -49,6 +49,7 @@ olad_libolaserver_la_SOURCES = $(ola_server_sources) \
                                olad/OlaServer.cpp \
                                olad/OlaDaemon.cpp
 olad_libolaserver_la_CXXFLAGS = $(COMMON_CXXFLAGS) \
+                                $(COMMON_PROTOBUF_CXXFLAGS) \
```

Comment:
You can just swap to COMMON_PROTOBUF_CXXFLAGS, as COMMON_CXXFLAGS is just a subset of the flags, with less skipping of errors.